.github/workflows/build.yml

```
@@ -1,109 +1,123 @@
-    - name: Install dependencies
-      run: pnpm install --frozen-lockfile
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
 
-    - name: Try to build the packages
-      run: pnpm build:packages
+      - name: Run unit tests
+        run: pnpm test
+        env:
+          REDIS_HOST: localhost
+          REDIS_PORT: ${{ job.services.redis.ports[6379] }}
```

actionlint (1.7.12)

[error] property access of object must be type of string but got "number"

(expression)